test/unit/test_aiakos.yml

```diff
@@ -0,0 +1,12 @@
+#----------------------------------------------------------------------------
+# Aiakos application section (mandatory)
+#----------------------------------------------------------------------------
+app:
+  port: 3000		# App listen port
+
+#-----------------------------------------------------------------------------
+#  openstack admin (mandatory)
+#-----------------------------------------------------------------------------
+openstack:
+  username: ''		# admin openstack user
+  password: ''		# admin openstack password
```

Comment:
this file is only used by test_config.js, in order to test, how to config.js reads the file.